utils/staking-miner/src/info.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+use sp_version::RuntimeVersion;
+
+#[derive(Debug, serde::Serialize)]
+pub(crate) struct Info {
```

Comment:
`Info` is a bit vague, I would rather call it `RuntimeVersion`or something like that.

Even if it's doesn't have all the data that the `RuntimeVersion` from substrate.

Reply:
Indeed, I wanted to  it vague and extensible but I don't see an immediate need for that. I will make it more specific.

Reply:
So I left the subcommand as `info` since it returns several information and no longer one version but internally the `struct` are now named more specifically.